help/help.go

```diff
@@ -209,7 +209,7 @@ func (m Model) FullHelpView(groups [][]key.Binding) string {
 		out = append(out, col)
 
 		// Separator
-		if i < len(group)-1 {
+		if i < len(groups)-1 {
```

Comment:
In the case of columns containing two items, this essentially said `if i < 1, add the separator`. This led to only the 0th column receiving the separator, no matter how many other columns are displayed.